js/client-battle.js

```diff
@@ -403,9 +406,27 @@
 				break;
 			}
 		},
-		getTimerHTML: function () {
+		timerInterval: 0,
+		getTimerHTML: function (ticking) {
 			var time = 'Timer';
 			var timerTicking = (this.battle.kickingInactive && this.request && !this.request.wait && !(this.choice && this.choice.waiting)) ? ' timerbutton-on' : '';
+
+			if (!ticking) {
```

Comment:
Bad naming maybe; there were other bugs anyway so I renamed it while I was there.